sqlQueries/DropOverhaulTables.sql

```diff
@@ -10,4 +10,6 @@ DROP TABLE if EXISTS sharedfunds;
 
 DROP TABLE if EXISTS funds;
 
-DROP TABLE if EXISTS user;
\ No newline at end of file
+DROP TABLE if EXISTS user;
+
+DROP TABLE if EXISTS Session;
```

Comment:
What is this "Session" Table? I believe I heard about it being a result of one of the plug-ins we used. If so, which plug in is it? Also, assuming this keeps track of which computers are signed into an account, are we sure that we won't be accidentally ending all sessions when someone else signs in?

Reply:
I didn't realize this was being added to the commit, but it is useful. The Passport package we use for authentication requires a Session table. Specifically, upon initialization it will check if Session exists, and if not it will create one. Important: I got critical errors when using my local database because it had an unrelated view named Session.
I haven't looked into it, but I think it just tracks the most recent activity of a user so it can resume them later. This should not affect multiple user support, but we should test what happens if two users log in with the same credentials simultaneously.